client/src/components/HowYouCanHelp/tests/__snapshots__/howYouCanHelp.test.tsx.snap

```diff
@@ -30,7 +30,17 @@ exports[`rendering of the HowYouCanHelp checks if various text fields are visibl
         .
       </li>
       <li>
-        CEQ will publish a Request for Information on federalregister.gov.
+        CEQ will publish a Request for Information on 
```

Comment:
Nit: @KameronKerger is this the right language to use future tense here? Would it be more like `CEQ has published a Request...`